CONTRIBUTING.md

```diff
@@ -1,9 +1,11 @@
+# Contributing to Playful Programming Blog
```

Comment:
We generally want to discourage calling it a `blog`, especially since we publish books and are moving towards video and course content in the future. `platform` might be better